src/files/file-input/FileInput.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Button from '../../components/button/Button'
+
+export function readAsBuffer (file) {
```

Comment:
Lets move this data wrangling logic to the redux bundle. It's something we'd need to do regardless of what UI library we used, so it let's pull it out.